docs/content/preview/integrations/akka-ysql.md

```diff
@@ -24,4 +24,297 @@ type: docs
+
+Akka Persistence enables stateful akka-actors to persist their state so that it can be recovered when an actor is either restarted, such as after a JVM crash by a supervisor or a manual stop-start, or migrated in a cluster.
+The Akka Persistence R2DBC plugin allows you to use Postgres or Yugabyte as a backend for Akka Persistence and Akka Persistence Query.
+The following section illustrates how to run an Akka Persistence R2DBC based application with YugabyteDB.
+
+## Prerequisites
+
+To use the [Akka Persistence R2DBC plugin](https://doc.akka.io/docs/akka-persistence-r2dbc/current/overview.html), ensure that you have the following:
+
+- YugabyteDB up and running. Download and install YugabyteDB by following the steps in [Quick start](../../../../quick-start/).
+- Java Development Kit (JDK) 8, 11 or 17 installed. JDK installers for Linux and macOS can be downloaded from [Oracle](http://jdk.java.net/), [Adoptium (OpenJDK)](https://adoptium.net/), or [Azul Systems (OpenJDK)](https://www.azul.com/downloads/?package=jdk). Homebrew users on macOS can install using `brew install openjdk`.
+- [sbt](https://www.scala-sbt.org/1.x/docs/) is installed.
+
+The following example is inspired from the [akka-cassandra-demo](https://github.com/rockthejvm/akka-cassandra-demo) repository.
+
+## Create a Scala project
+
+1. Create a project using the following command:
+
+    ```scala
+    sbt new scala/hello-world.g8
+    ```
+
+    Press enter when you're prompted to name the application.
+
+1. Add the following code in `build.sbt` file from your project's home directory.
+
+    ```sh
+    cd hello-world-template
+    ```
+
+    ```scala
+    lazy val akkaVersion = "2.7.0"
+    
+    libraryDependencies ++= Seq(
+      "com.typesafe.akka" %% "akka-actor-typed" % akkaVersion,
+      "com.typesafe.akka" %% "akka-stream" % akkaVersion,
+      "com.typesafe.akka" %% "akka-persistence-typed" % akkaVersion,
+      "com.typesafe.akka" %% "akka-serialization-jackson" % akkaVersion,
+      "com.typesafe.akka" %% "akka-cluster-tools" % akkaVersion,
+      "ch.qos.logback" % "logback-classic" % "1.2.11",
+      "com.lightbend.akka" %% "akka-persistence-r2dbc" % "1.0.1")
+    
+    //Add the yugabyte java driver to the dependencies of the application.
+    libraryDependencies ++= Seq("com.yugabyte" % "java-driver-core" % "4.6.0-yb-11")
+    
+    //Exclude the datastax java driver from getting pulled due to nested dependency.
+    excludeDependencies ++= Seq(
+      ExclusionRule("com.datastax.oss", "java-driver-core"))
+    
+    run / fork := true
+    Global / cancelable := false // ctrl-c
+    ```
+
+## Use the Akka Persistence R2DBC plugin
+
+To create an application using the plugin, do the following:
+
+### Write a sample application
+
+To write a sample application and customize its configuration, do the following:
+
+1. Copy the following code in `src/main/scala/Main.scala` as follows:
+
+    ```scala
```

Comment:
This is the exact same source code as for the Cassandra sample. Differences in build dependencies and configuration.